tests/e2e/utils/flows.js

```diff
@@ -47,6 +47,7 @@ const WC_SUBSCRIPTIONS_PAGE =
 	baseUrl + 'wp-admin/edit.php?post_type=shop_subscription';
 const ACTION_SCHEDULER = baseUrl + 'wp-admin/tools.php?page=action-scheduler';
 const WP_ADMIN_PAGES = baseUrl + 'wp-admin/edit.php?post_type=page';
+const WP_THEMES_PAGE = baseUrl + 'wp-admin/themes.php';
```

Comment:
Can be removed in favor of `activateTheme` provided by `@wordpress/e2e-test-utils`

Reply:
Removed in https://github.com/Automattic/woocommerce-payments/pull/8164/commits/8a7bbc244edff7f1723e103975ca10733fb3e23b. the suggestion!